test/unit/superstring.test.js

```diff
@@ -52,5 +52,12 @@ describe('String utilities', function () {
                 generatorSet[extension] = name;
             });
         });
+
+        it('should not include generators that are incomplete', function () {
```

Comment:
How it's different from the first assertion `should contain required generators`?
And how incomplete generators like `system.directoryPath` spilled through it?

Reply:
`FakerJS` has a `directoryPath` generator which is marked TODO and returns nothing. Because it _is_ a function, it does not get caught in `.to.be.a('function')`.